src/main/java/org/broadinstitute/dsm/util/ParticipantDataUtil.java

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.dsm.util;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import org.broadinstitute.dsm.db.dao.participant.data.ParticipantDataDao;
+import org.broadinstitute.dsm.db.dto.participant.data.ParticipantDataDto;
+import org.broadinstitute.dsm.model.participant.data.FamilyMemberConstants;
+
+import java.util.List;
+import java.util.Map;
+
+public class ParticipantDataUtil {
+
+    private static final Gson gson = new Gson();
+
+    public static boolean hasProbandEmail(String participantDataId, String collaboratorParticipantId) {
```

Comment:
As far as I remember there is class `ParticipantUtil` already, you could use that class, other than that, personally don't like the idea having method as utility method and this method has DB interaction in it.